src/components/users/index.js

```diff
@@ -235,6 +316,76 @@ class Users extends React.Component {
                         }
                       </BootstrapModal.Footer>
                     </BootstrapModal>;
+
+                  case 'inviteUser':
+                    return <BootstrapModal className='create-key-pair-modal' show={this.state.modal.visible} onHide={this.closeModal.bind(this)}>
+                      <BootstrapModal.Header closeButton>
+                        <BootstrapModal.Title>Invite a New User</BootstrapModal.Title>
+                      </BootstrapModal.Header>
+
+                      <BootstrapModal.Body>
+                        <form>
+                        <p>Creating an invitation is the way to get new people onto this installation.</p>
+                        <p>You can chose whether or not to send the invitee an email, and which organization the invitee should become a member of once they accept their invitation.</p>
```

Comment:
> You can chose whether or not to send the invitee an email

That part is really obvious from the checkbox down there. It should also be the case for 99% of invites. So I'd remove this.

> and which organization the invitee should become a member of

This is also really obvious. I'd remove the paragraph as a whole.